src/components/geolocation/GeolocationDirective.js

```diff
@@ -35,8 +37,34 @@
         var view = map.getView();
         var accuracyFeature = new ol.Feature();
         var positionFeature = new ol.Feature(new ol.geom.Point([0, 0]));
+        var headingFeature = new ol.Feature(new ol.geom.Point([0, 0]));
+        var headingStyleFunction = function(rotation) {
+          return new ol.style.Style({
+            image: new ol.style.Icon({
+              anchorXUnits: 'fraction',
+              anchorYUnits: 'fraction',
+              opacity: 1,
+              rotation: rotation,
+              src: 'components/geolocation/style/orientation_8.png'
+            })
+          });
+        };
+        var shapeFeature = new ol.Feature({
```

Comment:
Yes, I implemented both in order to compare the alternatives. I'll remove this, since I go for png-way.